### ISSUE TITLE
Weird URL for accounts -> recently active

### ISSUE THREAD

**Issue by kesselb** [nextcloud/server#46776]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The URL for the "Recently active" list in the accounts manager is `/settings/users/__nc_internal_recent`.

It works and it's no problem form a technical point.

Only a bit weird, given that we use cleaner urls for admins and disabled accounts.

Admins: `/settings/users/admin`
Disabled accounts: `/settings/users/disabled`

### Steps to reproduce

1. Login as admin
2. Go to accounts management
3. Click "Recently active"
4. Notice url

### Expected behavior

A clean url ;)

cc @sorbaugh @Pytal

**Comment by Pytal** [comment]
Yes this is intentional, initial idea was to use `recents` but since this would be breaking went with this id suggested by @susnux

**Comment by kesselb** [comment]
I see, to avoid conflicts with existing groups. 

The id for the disabled accounts section should be changed.

If you add a group and call it "disabled", the section but also the group disappears :see_no_evil: 

[Screencast from 2024-07-28 19-57-12.webm](https://github.com/user-attachments/assets/4e2b6465-1fa2-4bfd-9e29-68f535b8a2fb)

I'm reopening the issue for now, please let me know if you want me to log a fresh issue.

**Comment by susnux** [comment]
I think this can be solved by do not use fake groups but provide the groups as special lists.
And the URLs could also just be adjusted to look nice by changing them to e.g.
`/settings/users/recent` -> recent or disabled etc
`/settings/users/group/realGroupId` -> for real groups 

(or similar)

**Comment by joshtrichards** [comment]
> If you add a group and call it "disabled", the section but also the group disappears 🙈

Made me think of #36424.

Also some similar issues in the OCS routes apparently: #36492 (see "Bonus" section).

Also this reminds me how our inconsistent use of gid versus displayname is leading to bugs like #37613, #36840, #50286
